### ISSUE TITLE
Consider limiting keywords to 3 per paper?

### ISSUE THREAD

**Issue by fredhohman** [gtvis/gtvis.github.io#3]
Since the papers page is so long (the filters are great!), I think keeping the number of tags to a max, e.g., 3, would help reduce information overload.

**Comment by arpitnarechania** [comment]
Good point -- the scraper that pre-loaded keywords for a few papers _may_ have included keywords from authors as well as the publishers. 

Now authors generally put 1-5 keywords in the manuscript itself hence clipping to 3 might be harsh? As an alternative, I propose:
- we manually sift through the papers that have keywords today and just keep the author keywords (by looking up the manuscripts)
- mention *author* keywords/ tags only in the README to ensure limited count going forward.

Thoughts?

**Comment by fredhohman** [comment]
Good suggestions. Many times keywords include "Visualization" or "Visual Analytics" but I think this would be a bit redundant for this site, since, well, every paper is related to visualization. That's why I went with slightly more informative (but not too specific) keywords. Whether it's 3 or 5 doesn't bother me, but I'd say more than 5 becomes too overloaded on the page. One suggestion is to use light gray background and darker gray text instead of the bright and bold GT gold for the keyword tags. I do like them, I just think the problem is when there are a bunch, and the current design draws my eye towards them and not the paper title!
